pages_unit_tests/operator-address.test.js

```diff
@@ -38,9 +38,7 @@ describe("<OperatorAddress />", () => {
 
     it("renders correct header", () => {
       const header = wrapper.find(Heading);
```

Comment:
Needs to test the alternative of the conditional as well, when role isn't partnership
Need to add similar tests to address-select and address-manual to test conditional headings.